LICENSE

```diff
@@ -0,0 +1,25 @@
+BSD 2-Clause License
+
+Copyright 2020-2021, Caesar authors & contributors
+All rights reserved.
```

Comment:
```suggestion
```
same goes for this phrase

Reply:
I'm happy to remove it but the [other](https://github.com/Homebrew/brew/blob/master/LICENSE.txt) [BSD 2-clause](https://github.com/ponylang/ponyc/blob/master/LICENSE) [repos](https://github.com/go-redis/redis/blob/master/LICENSE) that I see all include this verbiage from the template script. 

Is there any particular reason to remove this language other than it being basically meaningless? Leaving it unmodified seems simpler and more fool-proof to me as a non-expert.

Reply:
hmm, I don't see that language on wikipedia or opensource.org
https://en.wikipedia.org/wiki/BSD_licenses#2-clause_license_("Simplified_BSD_License"_or_"FreeBSD_License")
https://opensource.org/licenses/BSD-2-Clause
weird to me that most repos have it but those without also work with the license detector
https://github.com/cyfdecyf/cow
https://github.com/insidegui/WWDC

Reply:
OK it's clearly not necessary. I'll remove it. :+1: